composer.json

```diff
@@ -65,6 +65,9 @@
         "oneup/flysystem-bundle": "^3.0",
         "m2mtech/flysystem-stream-wrapper": "^1.0 (Required when using Flysystem)"
     },
+    "conflict": {
+        "symfony/phpunit-bridge": "<6.0.17"
```

Comment:
You can directly require the minimum `^6.0.17`.

Reply:
Right, much better!